### PR TITLE
Reintroduce "new notebook" menu item inside notebook

### PR DESCRIPTION
**Pull request by wlach**

### CODE REVIEW

**Comment by wlach** [comment]
i.e. this: 
image

**Comment by wlach** [comment]
@bcolloran I think the redux changes in this PR might be more up your wheelhouse to review.

File: src/eval-frame/iodide-api/__test__/environment.test.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import { environment } from '../environment'
 import { store } from '../../store'
-import { newNotebook } from '../../actions/actions'
+import { resetNotebook } from '../../../actions/actions'
```
Comment (bcolloran) [inline src/eval-frame/iodide-api/__test__/environment.test.js]: should import from '../../actions/actions'

File: src/eval-frame/iodide-api/__test__/evalQueue.test.js [hunk 2]
```diff
@@ -7,13 +7,14 @@ import {
   getRunningCellAsyncProcessStatus,
 } from '../evalQueue'
 import { store } from '../../store'
-import { temporarilySaveRunningCellID, newNotebook } from '../../actions/actions'
+import { temporarilySaveRunningCellID } from '../../actions/actions'
+import { resetNotebook } from '../../../actions/actions'
```
Comment (bcolloran) [inline src/eval-frame/iodide-api/__test__/evalQueue.test.js]: should import from '../../actions/actions'

File: src/eval-frame/iodide-api/__test__/environment.test.js (hunk 1 above)
Comment (bcolloran) [reply src/eval-frame/iodide-api/__test__/environment.test.js]: (that is `src/eval-frame/actions/actions.js`)

File: src/eval-frame/iodide-api/__test__/evalQueue.test.js (hunk 2 above)
Comment (bcolloran) [reply src/eval-frame/iodide-api/__test__/evalQueue.test.js]: (that is `src/eval-frame/actions/actions.js`)

**Comment by bcolloran** [comment]
@wlach i don't know if i'm doind something wrong, but i get unexpected behavior when i try the following:
1) in chrome, open localhost:8000 in a new incognito window (to make seure logins are cleared)
2) navigate to an existing notebook
3) click the "new notebook" menu item

i get a "save file" window with the filename pre-filled as "login", and "about:blank" appears in the url bar of the tab behind the "save file" window. Closing the "save file" window closes the "about:blank" tab, and no new notebook is available.

peek 2018-09-05 15-34

**Comment by wlach** [comment]
@bcolloran hrm, it looks like the content-type is getting set incorrectly for the `/login/` endpoint to `<WSGIRequest: GET '/login'>`. It seems like Firefox renders it as html anyway, but Chrome doesn't. This problem doesn't have to do with this PR, so I think we should just go ahead and merge it if you don't see anything else wrong. I have another PR working to use a shorthand to render our templates, which seems to set the mimetypes correctly

screen shot 2018-09-06 at 5 13 35 pm

**Comment by bcolloran** [comment]
ok @wlach sounds good, i'll merge.

**Pull request merged**